src/main/java/jchess/gui/presenter/mainwindow/IMainPresenter.java

```diff
@@ -6,4 +6,5 @@
 
 public interface IMainPresenter extends IPresenter {
 	public void addTab(Component oComponent, String stName);
+	public void closetab();
```

Comment:
Since you need to revert changes in ExtendedRuleProcessor, I mentioned at the bottom of this review, kindly make correction here as well.
Please make 't' of Tab captial i.e. closeTab();